### ISSUE TITLE
Action Bar - expand/collapse buttons missing text equivalents

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#710]
### Expected behavior

Text equivalents for icons present
### Actual behavior

No text available in component for button labels (e.g., Expand all ..., Collapse all..).

**Comment by Blackbaud-PatrickOFriel** [comment]
We can fix that, but it's just a problem with the demo, not the action bar as a whole. Any of those arbitrary action bar items can have a `bb-action-bar-item-label` applied which will be available for screen readers.

**Comment by Blackbaud-MattGregg** [comment]
Doesn't seem like it would be worthwhile to fix the demo, but this brings up a few things in my mind as I can see that both FE NXT and RE NXT are using this as is and missing text equivalents.  RE NXT looks to have an Angular a11y module writing something in the button element -  ng-attr-aria-label="{{bbActionBarItem.bbActionBarItemLabel}}" 
- How to do others know this label is available as that doesn't seem to be in the documentation as a setting?
- If this is the way to solve this, what about having default values in the component and then implementing teams can update based on context? 
- Maybe even making these attributes required? While I could see making this required, I'm not sure if that's a good approach given there would be other ways to include text equivalents.

**Comment by Blackbaud-PatrickOFriel** [comment]
It's right in the documentation under `bb-action-bar-item-label`. We should fix the actual demo code to use this. There should not be a default value because you only add that label when you are not putting text in your button. Same reasoning for why it should not be required.

**Comment by Blackbaud-PatrickOFriel** [comment]
Are you looking at the correct docs site at https://skyux.developer.blackbaud.com/components/actionbar/?

**Comment by Blackbaud-MattGregg** [comment]
Doh. Somehow I linked to http://blackbaud-skyux-prod-ux-beta.azurewebsites.net/components/actionbar/

I see what you're saying looking at this now. The expand/collapse are buttons just like the others. Then it's a matter of how to get the teams to think about this during implementation. Having in the demo then could be useful as an example. thx.

**Comment by Blackbaud-PatrickOFriel** [comment]
In [v1.6.8](https://github.com/blackbaud/skyux/releases/tag/v1.6.8)
